src/Appwrite/Utopia/Database/Validator/Queries.php

```diff
@@ -87,6 +87,7 @@ public function isValid($value): bool
                 case Query::TYPE_GREATER:
                 case Query::TYPE_GREATEREQUAL:
                 case Query::TYPE_SEARCH:
+                case Query::TYPE_SLEEP:
```

Comment:
Why do we add this here? this is a validator for external input, we shouldn't allow this as an operator that users can pass.

Reply:
removed!